modules/users/manifests/eamonnmcelroy.pp

```diff
@@ -0,0 +1,8 @@
+# Creates the eamonnmcmrloy user
```

Comment:
Just got one last misspelling to fix here.

Reply:
ARGHHH, Eamonn actually sent me this and seemingly spelled his name doubly wrong haha -'eamonnmcmrloy' -> 'eamonnmcelroy'